docs/content/guides/recipes/Terraform/howto-private-registry/snippets/env.bicep

```diff
@@ -0,0 +1,61 @@
+import radius as radius
+
+//ENV
+resource env 'Applications.Core/environments@2023-10-01-preview' = {
+  name: 'prod'
+  properties: {
+    compute: {
+      kind: 'kubernetes'
+      resourceId: 'self'
+      namespace: 'default'
+    }
+    recipeConfig: {
+      terraform: {
+        authentication:{
+          git:{  
+            // PAT is a required key value, personal access token
+            pat:{
+              // This has to be a path name to the secret store
```

Comment:
This comment is at the wrong location. It makes it seem like `dev.azure.com` is the path name to the secret store.

Reply:
Also it's secretStore ID, not "path name"